examples/hello-world/hello-numpy-cross-val/job.py

```diff
@@ -0,0 +1,171 @@
+from nvflare.app_common.workflows.scatter_and_gather import ScatterAndGather
+from nvflare.recipe import SimEnv
+
+SERVER_MODEL_DIR = "/tmp/nvflare/server_pretrain_models"
+CLIENT_MODEL_DIR = "/tmp/nvflare/client_pretrain_models"
+WORKSPACE_DIR = "/tmp/nvflare/jobs/workdir"
+
+
+def define_parser():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--n_clients", type=int, default=2)
+    parser.add_argument(
+        "--mode",
+        type=str,
+        default="pretrained",
+        choices=["pretrained", "training"],
+        help="Mode: 'pretrained' for standalone CSE, 'training' for training+CSE",
+    )
+    parser.add_argument("--num_rounds", type=int, default=1, help="Number of training rounds (for training mode)")
+    return parser.parse_args()
+
+
+def run_pretrained_cse(n_clients: int):
+    """Run standalone cross-site evaluation with pre-trained models using Recipe API."""
+    print("\n=== Running Cross-Site Evaluation with Pre-trained Models ===\n")
+
+    # Cross-site evaluation with pre-trained models
+    recipe = NumpyCrossSiteEvalRecipe(
+        name="hello-numpy-cse",
+        min_clients=n_clients,
+        model_locator_config={
+            "model_dir": SERVER_MODEL_DIR,
+            "model_name": {"server_model_1": "server_1.npy", "server_model_2": "server_2.npy"},
+        },
+        client_model_dir=CLIENT_MODEL_DIR,
+    )
+
+    # Run in simulation environment
+    env = SimEnv(num_clients=n_clients)
+    run = recipe.execute(env)
+
+    print()
+    print("Cross-site evaluation complete!")
+    print("  Result location:", run.get_result())
+    print("  Job status:", run.get_status())
+    print()
+    print("To view results:")
+    print(f"  cat {run.get_result()}/cross_site_val/cross_val_results.json")
+    print()
+
+
+def run_training_and_cse(n_clients: int, num_rounds: int):
+    """Run FedAvg training followed by cross-site evaluation using FedJob API."""
+    print("\n=== Running Training + Cross-Site Evaluation ===\n")
+    print(f"Configuration: {n_clients} clients, {num_rounds} training rounds\n")
+
+    # Create FedJob with both training and CSE workflows
+    job = FedJob(name="hello-numpy-train-cse", min_clients=n_clients)
+
+    # Server components
+    persistor_id = job.to_server(NPModelPersistor())
+    aggregator_id = job.to_server(InTimeAccumulateWeightedAggregator(expected_data_kind=DataKind.WEIGHTS))
+    shareable_generator_id = job.to_server(FullModelShareableGenerator())
+    model_locator_id = job.to_server(NPModelLocator())  # Will locate models from persistor
+    formatter_id = job.to_server(NPFormatter())
+    job.to_server(ValidationJsonGenerator())
+
+    # Workflow 1: Training (ScatterAndGather)
+    controller = ScatterAndGather(
+        min_clients=n_clients,
+        num_rounds=num_rounds,
+        persistor_id=persistor_id,
+        aggregator_id=aggregator_id,
+        shareable_generator_id=shareable_generator_id,
+    )
+    job.to_server(controller)
+
```

Comment:
can we either 
(1) make a FedAvgWithCrossSiteEval recipe
(2) put these in CrossSiteEval recipe, and have an argument to include / not include this part to accommadate different use cases